spesifikasjoner/afpant/afpant-oppnaaddprioritet/oppnaaddprioritet.md

```diff
@@ -0,0 +1,56 @@
+# Melding om oppnådd prioritet (OppnaaddPrioritetOk)
+'OppnaaddPrioritetOk' er en meldingstype som sendes fra megler til bank for å informere om at avtalt prioritet er oppnådd. 
+Meldingen kommer etter meldingen 'GjennomfoertEtinglysing'(tinglyst hos Kartverket) som forventes mottatt først. 'GjennomfoertEtinglysing' skal sendes så fort tinglysning er gjennomført.
```

Comment:
Det er vel strengt tatt ikke noe avhengighet mellom `GjennomfoertEtinglysing` og `OppnaaddPrioritetOk` ... ?

Reply:
@ropatambitadotcom  jo, siden oppnådd prioritet bare er ett flagg trenger man 'GjennomfoertEtinglysing' for å få dokumentnummer, data osv. Dette er informasjon man trenger på bank-siden for å kunne slette osv senere